lib/libutils/ext/include/confine_array_index.h

```diff
@@ -142,4 +142,11 @@ static inline size_t confine_array_index(size_t index, size_t size) {
   return safe_index;
 }
 #endif
+#ifdef __riscv
+static inline size_t confine_array_index(size_t index, size_t size) {
+	(void)index;
+	(void)size;
+	return index;
+}
```

Comment:
Please use `__unused` instead of `(void)size;`, and note that `index` is used.
```C
static inline size_t confine_array_index(size_t index, size_t size __unused) {
	return index;
}
```